DOC-2321 updated full-featured demo `example.js` to match and fix typos in `index.js`

FarzadHayat opened the pull request:
Ticket: DOC-2321

Site: [DOC-2321 site](http://docs-feature-70-doc-2321.staging.tiny.cloud/docs/tinymce/latest/full-featured-premium-demo/)

Follow-up PR to make some improvements to the full-featured with premium plugins demo.

Changes:
* updated full-featured demo `example.js` to match and fix typos in `index.js`

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`

Review:
- [x] Documentation Team Lead has reviewed

MitchC1999 commented on modules/ROOT/examples/live-demos/full-featured/example.js:
```diff
@@ -148,5 +148,6 @@ tinymce.init({
     }
-  ]
+  ],
+  revisionhistory_fetch: fetch_revisions,
```
This is undefined. This option needs to be implemented correctly for users who just copy paste.

FarzadHayat replied:
[previous code](https://github.com/tinymce/tinymce-docs/pull/3161/files#diff-c285688d2a88fec7c645a37f976828932daa9360a988e4c83f6946ed69a7e74eR115-R118) in `example.js` contains other incomplete function stubs. I have added a comment in [363e718](https://github.com/tinymce/tinymce-docs/pull/3161/commits/363e7180e507f6d73b8120da7ea433e11f567cea) to make it obvious that the user needs to implement these functions. however, I would agree it makes sense to include at least a basic implementation of the function so that users who want to copy-paste the code have an easier time getting started.

FarzadHayat replied:
@MitchC1999 [previous code](https://github.com/tinymce/tinymce-docs/pull/3161/files#diff-c285688d2a88fec7c645a37f976828932daa9360a988e4c83f6946ed69a7e74eR115-R118) in `example.js` contains other incomplete function stubs. I have added a comment in [363e718](https://github.com/tinymce/tinymce-docs/pull/3161/commits/363e7180e507f6d73b8120da7ea433e11f567cea) to make it obvious that the user needs to implement these functions. however, I would agree it makes sense to include at least a basic implementation of the function so that users who want to copy-paste the code have an easier time getting started.

FarzadHayat replied:
even something like `revisionhistory_fetch: () => Promise.resolve([])` is enough.

MitchC1999 commented on modules/ROOT/examples/live-demos/full-featured/example.js:
```diff
@@ -3,7 +3,7 @@ const isSmallScreen = window.matchMedia('(max-width: 1023.5px)').matches;
 tinymce.init({
   selector: 'textarea#full-featured',
-  plugins: 'preview powerpaste casechange importcss tinydrive searchreplace autolink autosave save directionality advcode visualblocks visualchars fullscreen image link media mediaembed codesample table charmap pagebreak nonbreaking anchor tableofcontents insertdatetime advlist lists checklist wordcount tinymcespellchecker a11ychecker editimage help formatpainter permanentpen pageembed charmap ti ...
+  plugins: 'preview powerpaste casechange importcss tinydrive searchreplace autolink autosave save directionality advcode visualblocks visualchars fullscreen image link media mediaembed codesample table charmap pagebreak nonbreaking anchor tableofcontents insertdatetime advlist lists checklist wordcount tinymcespellchecker a11ychecker editimage help formatpainter permanentpen pageembed charmap ti ...
```
If we are including the export/import plugins, don't we need to specify the service url options as well?

FarzadHayat replied:
Removed these plugins from the demo and added them to the excluded plugins section on the demo page. [363e718](https://github.com/tinymce/tinymce-docs/pull/3161/commits/363e7180e507f6d73b8120da7ea433e11f567cea)

MitchC1999 commented on modules/ROOT/examples/live-demos/full-featured/example.js:
```diff
@@ -18,7 +18,7 @@ tinymce.init({
   },
   menubar: 'file edit view insert format tools table tc help',
-  toolbar: "undo redo | aidialog aishortcuts | blocks fontsizeinput | bold italic | align numlist bullist | link image | table media pageembed | lineheight  outdent indent | strikethrough forecolor backcolor formatpainter removeformat | charmap emoticons checklist | code fullscreen preview | save print | pagebreak anchor codesample footnotes mergetags | addtemplate inserttemplate | addcomment sho ...
+  toolbar: "undo redo | revisionhistory | aidialog aishortcuts | blocks fontsizeinput | bold italic | align numlist bullist | link image | table media pageembed | lineheight  outdent indent | strikethrough forecolor backcolor formatpainter removeformat | charmap emoticons checklist | code fullscreen preview | save print | pagebreak anchor codesample footnotes mergetags | addtemplate inserttemplat ...
```
We should include exportword exportpdf and importword here too

FarzadHayat replied:
no longer including as plugins have been removed from the demo

ltrouton reviewed (commented):
`exportword importword exportpdf` won't work without the service URL options provided so I am not sure if there is much value adding them to the demos now that I think about it. Maybe I am missing something?

FarzadHayat commented:
> `exportword importword exportpdf` won't work without the service URL options provided so I am not sure if there is much value adding them to the demos now that I think about it. Maybe I am missing something?

Removed these plugins from the demo and added them to the excluded plugins section on the demo page. [363e718](https://github.com/tinymce/tinymce-docs/pull/3161/commits/363e7180e507f6d73b8120da7ea433e11f567cea)
